charts/images.yaml

```diff
@@ -90,10 +90,6 @@ images:
   sourceRepository: github.com/gardener/vpn2
   repository: eu.gcr.io/gardener-project/gardener/vpn-seed-server
   tag: "0.4.0"
-- name: konnectivity-server
-  sourceRepository: github.com/gardener/replica-reloader
```

Comment:
I have opened https://github.com/gardener/replica-reloader/issues/7 to follow up after this PR is merged.